_content/papers/vittaka_analayo.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Vittaka"
+authors:
+ - analayo
+editor: "W. G. Weeraratne"
+booktitle: "Encyclopaedia of Buddhism"
+volume: 8
+number: 3
+pages: "713--716"
+publisher: "The Department of Buddhist Affairs"
+address: "Sri Lanka"
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/encyclopedia-entries/vitakka.pdf"
+year: 2009
+drive_links:
+ - "https://drive.google.com/file/d/1OoNIitHbYUGuN20HB_FDFj--etlvo4xR/view?usp=sharing"
+course: thought
+tags:
+  - jhana
+  - imagery
+  - ethics
+  - reasoning
+---
+
+A summary of vittaka (reasoning), with special attention to its ethical perspective, pschological analysis, role in the jhanas, and the various imageries used to explain the term.
```

Comment:
`pschological analysis` -> `psychology` perhaps?  `images` instead of `imageries`?